packages/compass-indexes/src/components/type-column/type-column.jsx

```diff
@@ -22,70 +39,50 @@ class TypeColumn extends PureComponent {
     openLink: PropTypes.func.isRequired,
   };
 
-  _textTooltip() {
+  canRenderTooltip() {
+    return (
+      ['text', 'wildcard', 'columnstore'].indexOf(this.props.index.type) !== -1
+    );
+  }
+
+  renderTooltip() {
     const info = pick(this.props.index.extra, [
       'weights',
       'default_language',
       'language_override',
       'wildcardProjection',
       'columnstoreProjection',
     ]);
-    return map(info, (v, k) => {
-      return format('%s: %j', k, v);
-    }).join('<br />');
-  }
-
-  /**
-   * Render the type div.
-   *
-   * @returns {React.Component} The type div.
-   */
-  renderType() {
-    let tooltipOptions = {};
-    if (
-      this.props.index.type === 'text' ||
-      this.props.index.type === 'wildcard' ||
-      this.props.index.type === 'columnstore'
-    ) {
-      const tooltipText = `${this._textTooltip()}`;
-      tooltipOptions = {
-        'data-tip': tooltipText,
-        'data-for': TOOLTIP_ID,
-        'data-effect': 'solid',
-        'data-multiline': true,
-        'data-border': true,
-      };
-    }
-
-    const helpLink = getIndexHelpLink(this.props.index.type.toUpperCase());
-
-    return (
-      <div
-        {...tooltipOptions}
-        className={styles[`type-column-property-${this.props.index.type}`]}
-        data-test-id="index-table-type"
-      >
-        {this.props.index.type}
-        {helpLink && (
-          <InfoSprinkle
-            helpLink={helpLink}
-            onClickHandler={this.props.openLink}
-          />
-        )}
-      </div>
-    );
+    const items = map(info, (v, k) => {
+      return <Body>{format('%s: %j', k, v)}</Body>;
```

Comment:
Would be nice if we could remove dependency on `util` here as this is a node.js package that is only available in the browsers through polyfills and this will be in the way for our plans to eventually disable node modules in electron renderer